templates/zerver/integrations/git.md

```diff
@@ -1,31 +1,41 @@
+# Zulip Git integration
+
 Get Zulip notifications for your Git repositories!
 
+{start_tabs}
+
 1. {!create-an-incoming-webhook.md!}
 
 1. {!download-python-bindings.md!}
 
-1. {!create-channel.md!}
-
 1. {!change-zulip-config-file.md!}
 
-    You may also need to change the value of `STREAM_NAME`.
```

Comment:
Several version of this had been previously discussed.
I had proposed appending this to the above macro in 2 different ways, but neither was perfect, so Lauryn recommended using the sentence version, and Alya recommended improving it further, I have now moved it into the Configuration Options section, since it has a default value set in the script.